src/Spryker/Client/RabbitMq/Model/Consumer/Consumer.php

```diff
@@ -75,7 +75,9 @@ public function receiveMessages($queueName, $chunkSize = 100, array $options = [
             $finished = true;
         }
 
-        return $this->collectedMessages;
+        $this->channel->basic_cancel($consumerTag, true);
```

Comment:
please don't use `basic_cancel` as it could affect the BC and also some unexpected changes for `UNACK` messages